src/makeCalculator.js

```diff
@@ -37,7 +37,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    add(result, number) {
+      return result + number;
+    },
```

Comment:
1. you should work with one function parameter
2. use this
```suggestion
    add(value) {
      this.result += value;

      return this;
    },
```